boards/at89s51.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Please RENAME all JSON files to the original TARGET NAMEs. Typically, make them UPPER CASE. We keep target devices in sync with vendor naming.

`boards/at89s51.json` -> `boards/AT89S51.json`